src/hw6/hw6.sql

```diff
@@ -0,0 +1,38 @@
+-- Command for Mariadb to use baseball database for the queries below.
+USE baseball;
+
+-- Fix the column type issue on it, so the id is sorting as integer.
+ALTER TABLE inning MODIFY COLUMN game_id INT UNSIGNED NOT NULL;
+
+-- Create intermediary table
+DROP TABLE IF EXISTS batter_game;
+
+CREATE TABLE batter_game
+SELECT bc.batter
+       , bc.game_id
+       , DATE(g.local_date) AS local_date
+       ,bc.atBat
+       ,bc.Hit
+FROM batter_counts bc
+JOIN game g
+ON bc.game_id = g.game_id;
+
+CREATE INDEX batter_game_index ON batter_game (batter, game_id);
```

Comment:
I think Julien said that you should have individual indexes on whatever field you are using in your group by. Since you're just using batter in your group by, you can probably get rid of game_id in your index.